src/reducers/scrollAnchor.js

```diff
@@ -74,6 +74,19 @@ export function scrollTo(state, scrollY) {
   const { rowOffsetIntervalTree } = state.getInternal();
   const { rowsCount } = rowSettings;
 
+  if (state.rowSettings.rowHeightGetter != undefined) {
+    // In case of variable row height, ask for the actual heights of the rows before the scroll position.
+    // Only for the ones that were not asked before
+    let { rowUntilOffsetsAreExact, rowOffsetIntervalTree } =
```

Comment:
here the rowOffsetIntervalTree is a duplicate import

Reply:
Ok. I removed this